tools/populate_tests.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Populates test history for a local BB org.
```

Comment:
Can you add something on how to use this script? I assume setting the given environment variables and then call ./tools/populate_tests.sh directly?